tee-worker/client-api/sidechain-api/package.json

```diff
@@ -6,7 +6,7 @@
     "version": "0.0.1-1",
     "scripts": {
         "clean": "rm -rf dist build node_modules",
-        "update-metadata": "../../bin/litentry-cli print-sgx-metadata-raw > prepare-build/litentry-sidechain-metadata.json",
+        "update-metadata": "../../bin/litentry-cli -U wss://localhost print-sgx-metadata-raw > prepare-build/litentry-sidechain-metadata.json",
```

Comment:
Can you also change the [WORKER_ENDPOINT](https://github.com/litentry/litentry-parachain/blob/dev/tee-worker/ts-tests/integration-tests/.env.local.example#L2) in integration-tests/env.local.example? I think it needs to be modified to wss://localhost. Thanks!